nanoFirmwareFlasher.Tool/Program.cs

```diff
@@ -36,14 +34,18 @@ public static async Task<int> Main(string[] args)
         {
             // take care of static fields
             _informationalVersionAttribute = Attribute.GetCustomAttribute(
-                Assembly.GetEntryAssembly()!,
+                typeof(Program).Assembly, // Cannot be Assembly.GetEntryAssembly()! as that fails in tests
```

Comment:
Code style calls for comments before the code line.

Reply:
Ok, did not remember that part.